sdk/eventhub/azure-eventhub/samples/async_samples/sample_code_eventhub_async.py

```diff
@@ -169,7 +172,7 @@ def example_create_async_eventhub_consumer_client():
 
     async def on_event(partition_context, event):
         logger.info("Received event from partition: {}".format(partition_context.partition_id))
-        # Do asynchronous ops on the received event
+        # Do some asynchronous ops on the received event
```

Comment:
same suggestion as keeping it consistent with the comment in sample codes

Reply:
Updated